# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../udev/udev-166-glibc-2.16-PRIO_PROCESS.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix to build with glibc >= 2.16

Without including <sys/resource.h> 'PRIO_PROCESS' would be undeclared.

--- udev-166/libudev/libudev-util-private.c.orig	2012-08-05 14:47:36.420234087 +0200
+++ udev-166/libudev/libudev-util-private.c	2012-08-05 14:56:13.022134963 +0200
@@ -21,6 +21,7 @@
 #include <grp.h>
 #include <sys/wait.h>
 #include <sys/param.h>
+#include <sys/resource.h>	/* PRIO_PROCESS */
 
 #include "libudev.h"
 #include "libudev-private.h"
--- udev-166/udev/udevd.c.orig	2012-08-05 14:53:23.997716697 +0200
+++ udev-166/udev/udevd.c	2012-08-05 14:55:46.630892576 +0200
@@ -43,6 +43,7 @@
 #include <sys/ioctl.h>
 #include <sys/inotify.h>
 #include <sys/utsname.h>
+#include <sys/resource.h>	/* PRIO_PROCESS */
 
 #include "udev.h"
 #include "sd-daemon.h"
